rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -1604,3 +1610,4 @@ def main():
 
 if __name__ == '__main__':
     sys.exit(main())
+
```

Comment:
Minor nit, can you kindly remove this space?

Reply:
removed, thanks!